mibidata/mibi_image.py

```diff
@@ -200,14 +213,28 @@ def fov_id(self):
     @fov_id.setter
     def fov_id(self, value):
         """Enforce consistency with folder."""
-        if value and not self.folder:
+        if not self.folder:
             self._folder = value
-        elif self.folder and value != self.folder.split('/')[0]:
+        elif value != self.folder.split('/')[0]:
             raise ValueError('fov_id must match folder, but here '
                              'folder={} and you are trying to set fov_id '
                              'to {}.'.format(self.folder, value))
         self._fov_id = value
 
+    @property
+    def aperture(self):
+        return self._aperture
+
+    @aperture.setter
+    def aperture(self, value):
+        if value is not None and value not in APERTURE_MAP.values():
+            raise ValueError(
+                'Invalid aperture code \'{}\', must use values \
+                    from the following map: {}'.format(value, APERTURE_MAP)
```

Comment:
Nit: You don't need the `\' or extra indentation for the line break inside the parentheses:
```
raise ValueError(
    'Invalide code .... '
    'from the following ...'
)

Reply:
🤯
Did not know that. Thanks.